backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -540,6 +540,38 @@ CREATE INDEX token_idx ON tokens (contract_index, contract_sub_index, token_id);
 -- Important for quickly filtering/sorting tokens by the order they were created by a contract.
 CREATE INDEX token_index_per_contract_idx ON tokens (token_index_per_contract);
 
+-- Relations between accounts and CIS2 tokens. Rows are added or updated in this table whenever a CIS2 `MintEvent`, `BurnEvent`
+-- or `TransferEvent` is logged by a contract claiming to follow the `CIS2 standard`.
+CREATE TABLE account_tokens (
+    -- An index/id for the row (row number).
+    index
+        BIGINT
+        PRIMARY KEY,
+  -- An account index (row in the `accounts` table) that owns a token balance.
+    account_index
+        BIGINT
+        NOT NULL,
+    -- The token index (row in the `tokens` table) of the associated token.
+    token_index
+        BIGINT
+        NOT NULL,
+    -- Every time a row is associated with a token, this index is incremented for that token.
+    -- This value is used for the `skip/take` parameter when querying token holders.
+    index_per_token
+        BIGINT
+        NOT NULL,
+    -- The accumulated balance of the token by the above account calculated by considering all `MintEvents`, `BurnEvents` and 
+    -- `TransferEvents` associated to the token and account. If no balance is specified when inserting a new row in the table,
+    -- the default balance 0 is used.
+    balance
+        NUMERIC
+        NOT NULL
+        DEFAULT 0,
+
+    -- Ensure that each token_index and account_index pair is unique.
+    CONSTRAINT unique_token_account_relationship UNIQUE (token_index, account_index)
```

Comment:
It seems that we never use the `index` column for this table, so we can get the constraint if we make these primary keys
```suggestion
    PRIMARY KEY (token_index, account_index)
```

Reply:
no longer relevant